CONTRIBUTING.md

```diff
@@ -61,15 +61,16 @@ The `bolt version-packages` command will generate a release commit, which will b
 The commands to run are:
 
 ```sh
-git checkout master
-git pull
-git branch -D temp-release-branch
-git checkout -b temp-release-branch
-bolt fresh
-bolt version-packages
-bolt format
-git add .
-git commit -m "Run version-packages"
+git checkout master && \
+git pull && \
+git branch -D temp-release-branch && \
+git checkout -b temp-release-branch && \
+bolt fresh && \
+bolt build && \
```

Comment:
NOTE: I added a `bolt build` command here to catch any issues that might otherwise only be caught in the publish phase (like in #1427)